sdk/include/psdk/winerror.h

```diff
@@ -3311,6 +3319,8 @@
 #define WINCODEC_ERR_WIN32ERROR                            _HRESULT_TYPEDEF_(0x88982f94)
 #define WINCODEC_ERR_INVALIDPROGRESSIVELEVEL               _HRESULT_TYPEDEF_(0x88982f95)
 
+#define MILERR_MISMATCHED_SIZE                             _HRESULT_TYPEDEF_(0x88980090)
```

Comment:
I'll check official Windows SDK to see whether they put the MIL/DWM errors also in winerror.h

Reply:
Yhile the DWM_*** errors above can indeed be found in Windows PSDK (7+, and perhaps Vista too), I cannot find this MILERR error (and others) in there, not even in Windows 10 PSDK.
I know that Wine added lots of these there too: https://source.winehq.org/patches/data/235259

Do you know whether these defines are actually new (like Windows 11+)? or just undocumented? (in which case they may have to go somewhere, like in dwmapi.h ...)

Reply:
I don't think they're particularly new, WINE puts them in winerror it seems. 

I don't mind putting them in dwmapi.h, I don't think we will sync this ever again.

Reply:
Actually yes you're correct.
I've found them in https://raw.githubusercontent.com/tpn/winsdk-10/master/Include/10.0.16299.0/shared/winerror.h for example (which means that github search is still broken and doesn't systematically search within files)